analyzer/main.zeek

```diff
@@ -284,7 +284,7 @@ function set_session(c: connection, message_id: int, opcode: LDAP::ProtocolOpcod
 }
 
 #############################################################################
-event protocol_confirmation(c: connection, atype: Analyzer::Tag, aid: count) &priority=5 {
+event analyzer_confirmation(c: connection, atype: Analyzer::Tag, aid: count) &priority=5 {
```

Comment:
The rename from `protocol_confirmation` to `analyzer_confirmation` happened only with zeek-4.2.0, so let's keep the existing handler for compatibility with the Zeek LTS release.

```suggestion
@if (Version::at_least("4.2.0"))
event analyzer_confirmation(c: connection, atype: Analyzer::Tag, aid: count) {
@else
event protocol_confirmation(c: connection, atype: AllAnalyzers::Tag, aid: count) {
@endif
```

This requires `@load base/misc/version`.